lib/handlers/lnurl_handler.dart

```diff
@@ -26,7 +27,7 @@ class LNURLHandler {
         return executeLNURLResponse(context, lnurlBloc, response);
       }
     }).onError((err) async {
-      final errorMessage = _getErrorMessage(texts, err.toString());
+      final errorMessage = _getErrorMessage(texts, err.toString()) ?? extractExceptionMessage(err);
```

Comment:
I suggest we pass error object to `_getErrorMessage(...)` and return `extractExceptionMessage` as it's default case instead of using nullish coalescing operator(**??**) here for sound null safety purposes.

Reply:
Nice suggestion, applied.